xpdan/tests/test_tiff_exporter.py

```diff
@@ -0,0 +1,14 @@
+from xpdan.glbl import an_glbl
+from xpdan.callbacks_core import XpdAcqLiveTiffExporter
+
+# standard config
+
+template = '/direct/XF28ID1/pe2_data/xpdUser/tiff_base/{start.sample_name}'
+data_fields = ['temperature', 'diff_x', 'diff_y', 'eurotherm'] # known devices
+
+
+tif_export = XpdAcqLiveTiffExporter('pe1_image', template, data_fields,
+                                    overwrite=True, db=an_glbl.exp_db)
+#alias
+def test_tiff_export_smoke_test():
+    an_glbl.exp_db.process(an_glbl.exp_db[-1], tif_export)
```

Comment:
Don't call the global db, just put `exp_db` in the signature of the function, also you should set up the config things in the test, or as a parametric test.
I think this might work better for you:
```python
def test_tiff_export_smoke_test(exp_db):
    tif_export = XpdAcqLiveTiffExporter('pe1_image', template, data_fields,
                                     overwrite=True, db=exp_db)
    exp_db.process(exp_db[-1], tif_export)
```